pkg/base/replicated_test.go

```diff
@@ -946,14 +953,14 @@ metadata:
     helm.sh/chart: postgresql-10.13.8
     app.kubernetes.io/instance: postgresql
     app.kubernetes.io/managed-by: Helm
-  namespace: 
+  namespace: ${POD_NAMESPACE}
```

Comment:
This and `TestGetenv` allow running tests in okteto.  This change is unrelated to the fix in the PR.